python_modules/dagster/dagster/__init__.py

```diff
@@ -266,12 +266,14 @@
 )
 from dagster._core.definitions.metadata import (
     BoolMetadataValue as BoolMetadataValue,
+    CodeReferencesMetadataValue as CodeReferencesMetadataValue,
```

Comment:
Seems like putting all this stuff in the top level include is jumping the gun here.